core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -11,15 +11,12 @@ class InstallGenerator < Rails::Generators::AppBase
 
     CORE_MOUNT_ROUTE = "mount Spree::Core::Engine"
 
-    LEGACY_FRONTEND = 'solidus_frontend'
-    DEFAULT_FRONTEND = 'solidus_starter_frontend'
```

Comment:
Up to you, but do you think it is ok to remove the constants (same for `DEFAULT_AUTHENTICATION` below)? It helps to check for string values that can be repeated multiple times (e.g., avoiding a typo that would create a mysterious bug).

Reply:
That's a good point, I'll try with a local variable first and fall back to a constant if that doesn't work

Context:
Those were removed because the role was a bit confusing, and with checks on the gemfile the default is actually dynamic (e.g. if you have the legacy one in the gemfile and `auto_accept` it will "default" to it, etc.).

Reply:
Cool. That's non-blocking, though :slightly_smiling_face: